script/gitlab_ci_run.sh

```diff
@@ -12,6 +12,15 @@
 
 source ./script/set.sh
 
+# inside the agc-container, the user is root and does not require sudo
+# to compatibility to other container, fake the missing sudo command
+if ! command -v sudo &> /dev/null
+then
+    cp ${CI_PROJECT_DIR}/script/gitlabci/fake_sudo.sh /usr/bin/sudo
+    chmod +x /usr/bin/sudo
+fi
+
+# TODO(sehrig): remove me
```

Comment:
Huh?

Reply:
It is a workaround for old HIP images. When I drop support for HIP 4.x I remove this part of code. Actual, I should extend the comment but the next PR is removing the old HIP container. Therefore, I would let it as it is.